apps/client/src/app/pages/portfolio/activities/import-activities-dialog/import-activities-dialog.scss

```diff
@@ -17,4 +17,16 @@
       }
     }
   }
+
+  .mat-stepper-horizontal {
```

Comment:
Please use the style classes `mt-1` / `mt-2` / `mt-3` / ... in the template instead writing these custom styles.